templates/zerver/help/import-users-and-channels-from-slack.md

```diff
@@ -0,0 +1,41 @@
+# Import Users and Channels from Slack
+
+{!follow-steps.md!} import users and channels from Slack to Zulip:
+
+1. Generate a Slack API token using Slack's [test token generator](https://api.slack.com/docs/oauth-test-tokens)
+   to import all of the necessary data.
+
+2. {!go-to-the.md!} **Your bots** {!settings.md!}
+
+3. Click on **Show/change your API key** button.
+
+4. Upon clicking the **Show/change your API key** button,
+   you will be asked to confirm your identity by entering
+   your password in the **Current password** field.
+
+5. Click the **Get API Key** button and copy generated API Key.
+
+6. Fill in all settings in `api/integrations/slack/zulip_slack_config.py`:
+
+    `SLACK_TOKEN` - token from point number 1.
```

Comment:
I did it but it transform sub-points into points not sure if it should be like that